deploy/catalog_resources/redhat/1.4/businessautomation-operator.1.4.0.clusterserviceversion.yaml

```diff
@@ -185,6 +185,18 @@ spec:
                 name: business-automation-operator
                 resources: {}
               serviceAccountName: business-automation-operator
+      clusterPermissions:
```

Comment:
this new role should be persisted in `csv-gen.go` for future csv generations via `make csv` ... e.g. -
https://github.com/kiegroup/kie-cloud-operator/blob/0c5fb7968959401ddecb50a843fed895c0e4c455/tools/csv-gen/csv-gen.go#L114
https://github.com/kiegroup/kie-cloud-operator/blob/0c5fb7968959401ddecb50a843fed895c0e4c455/tools/csv-gen/csv-gen.go#L298-L299